Mathlib/CategoryTheory/Limits/Shapes/Pullbacks.lean

```diff
@@ -2746,6 +2752,29 @@ def baseChange [HasPullbacks C] {X Y : C} (f : X ⟶ Y) : Over Y ⥤ Over X wher
     apply Over.OverMorphism.ext; apply pullback.hom_ext
     · dsimp; simp
     · dsimp; simp
-#align category_theory.limits.base_change CategoryTheory.Limits.baseChange
-
-end CategoryTheory.Limits
+#align category_theory.limits.base_change CategoryTheory.Over.baseChange
+
+-- deprecated on 2024-05-15
+@[deprecated] noncomputable alias Limits.baseChange := Over.baseChange
+
+/-- The adjunction `Over.map ⊣ baseChange` -/
+@[simps! homEquiv_apply homEquiv_symm_apply unit_app counit_app]
```

Comment:
As suggested by Adam, this should fix the issue:
```suggestion
@[simps! unit_app counit_app]
```

The verification "linting" that is done is a global verification at the scale of all mathlib, which can be very long as Kim said. However, it is possible to make part of these verification on a single file by inserting "#lint".

After this change, this PR should be ready to be merged. Then, you should edit the title of the PR as "feat(CategoryTheory): the adjunction between Over.map and baseChange" (following the conventions https://leanprover-community.github.io/contribute/commit.html)